CMSgov/mint-app
# [EASI-3666] FE Work for Updating Model Type Question

# EASI-3666

## Changes and Description

- Updated Model Type question in the basics overview page with multi-select checkboxes
- Updated Model Type question rendering in the Read Only page
- Added new `modelTypeOther` to all the queries
- Updated `modelType`'s translation mapping
- Added `modelTypeOther` to translation mapping
- Add `modelTypeOther` to csv export
- Add `modelTypeOther` to Filter group mapping
- Updated tests and snaps and mocks



## How to test this change

1. Navigate to Basic Overview task list form
2. Check that it matches the figma and is now a checkbox multi-select (instead of radio)
3. Fill out data 
4. Check to see if data presists, is recorded, shows up on Read Only and in CSV Export



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
